default.config.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Role: uninstalls
+# set home directory
+home_directory: /Users/
```

Comment:
you're right, they are only used once so I'm going to remove both `home_directory` and `root_directory` from this file and instead use "{{ ansible_env.HOME }}" and a static /usr/local for root